include/mgclient.h

```diff
@@ -15,7 +15,17 @@
 #ifndef MGCLIENT_MGCLIENT_H
 #define MGCLIENT_MGCLIENT_H
 
+#ifdef EMSCRIPTEN
+#define MGCLIENT_EXPORT
```

Comment:
Why not redirect `MGCLIENT_EXPORT` to `EMSCRIPTEN_KEEPALIVE`? I think they serve the same purpose.

Reply:
I really like this ;)